cmd/acra-connector/acra-connector.go

```diff
@@ -255,7 +253,11 @@ func main() {
 		os.Exit(1)
 	}
 
-	logging.Customize().SetFormat(*loggingFormat).SetServiceName(ServiceName).Complete()
+	// Start customizing logs here (directly after command line arguments parsing)
+	formatter := logging.CreateFormatter(*loggingFormat)
+	logging.SetServiceName(formatter, ServiceName)
```

Comment:
It's weird for **acra-connector** to be the only one without
```go
log.SetOutput(os.Stderr)
```
Is there a reason for that?

Reply:
no. Just my oversight, thank you